pullrequest.js

```diff
@@ -63,7 +71,7 @@ function toggleDiff(id, duration, display) {
         if (!useLocalStorage) {
             display = 'toggle';
         } else {
-            display = (localStorage.getItem(id) === 'collapse') ? 'expand' : 'collapse';
+            display = (localStorage.getItem(prId + id) === 'collapse') ? 'expand' : 'collapse';
```

Comment:
You have a function for getting the prId, should make a function called getDiffId or something so that we encapsulate all of these string concats...